Sources/SwiftSyntax/generated/raw/RawSyntaxNodesAB.swift

```diff
@@ -42,6 +42,14 @@ public struct RawAccessorBlockSyntax: RawSyntaxNodeProtocol {
       }
       return nil
     }
+    
+    public init(_ other: RawAccessorDeclListSyntax) {
+      self = .accessors(other)
+    }
```

Comment:
Why are these initializers needed? Couldn’t we just continue using the case names?

Reply:
True, I'll change the code generation logic so only enum cases with the payload type `Raw<Kind>Syntax` will get a convenience `init`.